doc/classes/OS.xml

```diff
@@ -262,6 +262,14 @@
 				[b]Note:[/b] This method is not supported on the Web platform. It returns an empty string.
 			</description>
 		</method>
+		<method name="get_entropy">
+			<return type="PackedByteArray" />
+			<param index="0" name="size" type="int" />
+			<description>
+				Retrieves a [PackedByteArray] of entropy with given [param size] from the operating system.
```

Comment:
Does "of given size" refer to the array size or the entropy? If it refers to the array size, what does "of entropy" refer to? Usually, I hear about the terms high/low entropy (or defined in a number of bits).

Reply:
In this case it refers to the entropy data used by the OS to generate random numbers, but unsure how to describe it here, maybe just "of entropy data" would be sufficient

Reply:
I agree this is quite confusing, I think we can just reuse what we have in generate_random_bytes:
```
Generates a PackedByteArray of cryptographically secure random bytes with given size.
```